### DISCUSSION TITLE
General designer UI tweaks

### DISCUSSION THREAD

**Discussion by benaunt** [Budibase/budibase#11359] [Ideas]
Hi,

When working with a larger application with many components on a screen I feel these small tweaks will help speed up UI creation when using the designer, especially when utilising side panels.

- The list of components that exist on the screen, either have tabs at the top of the components panel to filter, a drop down filter or split the components panel into individual icons on the far left navigation of the designer into smaller elements under a component header, potential filters listed below, when clicking an item under the filter it should display all child elements.
   - All components
   - Side panels 
   - Sections
   - Containers (as above)
   - Forms 
   - Data providers
   - Tables etc. 
 - When adding a new component, filters to each core component category
 
Thanks

**Comment by melohagan** [comment]
I had raised somewhat of a similar issue here as well! https://github.com/Budibase/budibase/issues/11124
